SU-SWS/decanter
# D8CORE-1329: removed the width 100% for xs breakpoint with buttons.

# READY FOR REVIEW

# Summary
- Removing the 100% width for buttons on XS screen

# Needed By (Date)
- Thursday

# Urgency
- Med

# Steps to Test

1. Look at the homepage on an XS screen. See the button with the centered text fill the whole screen.
2. Pull in and build Decanter.
3. Reload the page and see the button is at width: auto so that it only is as wide as the button text.
4. Verify the same on interior pages. 
2.  
2. Then this

# Affected Projects or Products
- Decanter
- D8CORE

# Associated Issues and/or People
- D8CORE-1329
- Other PRs
- @yvonnetangsu @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
